app/components/feedback_link/view.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if enabled %>
+<div class="govuk-inset-text">
```

Comment:
Do we / the formbuilder have a component for inset text?

Reply:
Not that I know of.